unified-codes/tools/scripts/dgraph/data/v2/schema.gql

```diff
@@ -0,0 +1,83 @@
+type Property {
+  type
+  value
+}
+
+type Category {
+  code
+  name
+  children
+  properties
+}
+
+type Product {
+  code
+  name
+  combines
+  children
+  properties
+}
+
+type Route {
+  code
+  name
+  children
+  properties
+}
+
+type DoseForm {
+  code
+  name
+  children
+  properties
+}
+
+type DoseQualification {
+  code
+  name
+  children
+  properties
+}
+
+type DoseStrength {
+  code
+  name
+  children
+  properties
+}
+
+type DoseUnit {
+  code
+  name
+  children
+  properties
+}
+
+type InnerPack {
+  code
+  name
+  children
+  properties
+}
+
+type PackSize {
+  code
+  name
+  children
+  properties
+}
+
+type OuterPack {
+  code
+  name
+  children
+  properties
+}
+
```

Comment:
Do we want to add `Manufacturer` and `Brand`, or can that wait?

Reply:
Good question, none of our current spreadsheet data has any Manufacturer or Brand details so I thought maybe best to revisit when they do start showing up?

Reply:
I thought that brand / manufacturer could wait - we don't have any information for them yet. Will need to support GTINs at some point, but there's just no data yet.

Reply:
OK, concensus seems to be no `Manufacturer` or `Brand` needed!